hugo.yaml

```diff
@@ -108,7 +108,7 @@ params:
 
   latest_engine_api_version: "1.47"
   docker_ce_version: "27.2.1"
```

Comment:
note to self: looks like we missed some updates here 🙈 

_starts working on a branch_